### DISCUSSION TITLE
Add profile name to title of window when using multiple profiles

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#831] [Ideas]
### What feature would you like to see?

Right now, I have two profiles, one named work and one named personal. However, they are a little annoying to keep track of since both the profiles look exactly the same in the dock. What I'm proposing is that the text here says "Zen Browser (Work)" or "Zen Browser (Personal)" to help distinguish the two. Ideally, these would maybe have a different color-coded icon, but I understand that is probably more work.

<img width="184" alt="Screenshot 2024-08-22 at 10 56 10 AM" src="https://github.com/user-attachments/assets/27b9e8a1-57f7-4fb5-b4b3-b0937a3c673d">
<img width="179" alt="Screenshot 2024-08-22 at 10 55 59 AM" src="https://github.com/user-attachments/assets/3816d511-3a94-46ee-a592-6954f6b173a8">

### What platform are you seeing the problem on?

macOS

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/450_

**Comment by willosbourne** [comment]
This would be a huge win for me in my book. It feels really clunky to not immediately know what Account I'm in, right now I'm just using light & dark modes for different accounts (since theme-ing has been... iffy so far in my experience), which is not necessarily a great experience.

**Reply by dam5s** [reply]
Did you know you can right click the UI and chose "change theme colors"? I was having the same issue with theme customization and discovered this option by accident, theming UI needs to be consolidated, but that "change theme colors" works pretty well for making profiles more identifiable.

![image](https://github.com/user-attachments/assets/7e6f8c97-1246-4737-9813-4d5d088adf0e)

**Reply by willosbourne** [reply]
I didn't know this, that does help things considerably. Thank you so much!

**Comment by call286** [comment]
That would be a great feature.
I'm using Zen in Linux, with KDE.
I use window rules to sort my windows to different activities and having the profile name in the window title would for example allow me to sort the "Zen - Work" window to the "Work" activity and the "Zen - Youtube" window to the "Private" activity on the second screen.

For my use case it would be sufficient to have the profile name added to the window title.

**Reply by bartier** [reply]
Have you found a workaround for it? I am looking for something similar but in macOS in my case

**Comment by renyuneyun** [comment]
Don't forget Zen is based on Firefox. This can be solved using an addon, such as [Window Titler](https://addons.mozilla.org/en-GB/firefox/addon/window-titler/)

**Reply by StudioNirin** [reply]
Unfortunately no, this doesnt work. Window titler seems to only take effect after the window has launched, so its too late to be effected by the kde special configs that trigger "at launch".
